src/experimental-layers/src/text-layer/font-atlas.js

```diff
@@ -1,9 +1,12 @@
 /* global document */
 import {Texture2D} from 'luma.gl';
 
+const GL_TEXTURE_WRAP_S = 0x2802;
+const GL_TEXTURE_WRAP_T = 0x2803;
+const GL_CLAMP_TO_EDGE = 0x812f;
```

Comment:
Why? You should use them from the gl or luma object.

Reply:
We are following this approach in other places, like here : https://github.com/uber/luma.gl/blob/a5c918b61415b187e0e541c30803c7df1e03762d/src/utils/typed-array-utils.js#L2
My understanding is, instead of importing all of the constants, this will reduce the code size, especially we are using only 3 constants here, @ibgreen  ?

Reply:
Yes if we use fields on an object, the minifier can't squash them. If we have local constants, the minifier replaces them with number.

Reply:
OK